fido/client.py

```diff
@@ -0,0 +1,21 @@
+def _twisted_web_client():
+    from twisted.web import client
+    return client
+
+
+class HTTP11ClientProtocolOverride(_twisted_web_client().HTTP11ClientProtocol):
```

Comment:
Right, but if someone imports `client.py`, then they'll get the reactor right? That test is only testing for `fido.py`. I'm not sure if this is an issue we should be concerned about. Perhaps a comment saying not to import the file pre-fork is sufficient?